### PR TITLE
Add client hooks for psycopg2

### LINKED ISSUE #29: Consider supporting postgres for tracing
In addition to mysql.

### PR DESCRIPTION
**Pull request by bitglue**
Fixes #29.

image

### CODE REVIEW

File: opentracing_instrumentation/client_hooks/psycopg2.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2015 Uber Technologies, Inc.
```
Comment (yurishkuro) [inline opentracing_instrumentation/client_hooks/psycopg2.py]: nit: 2017

**Comment by bitglue** [comment]
@yurishkuro Updated. Screenshot is in the description.

File: opentracing_instrumentation/client_hooks/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015 Uber Technologies, Inc.
+# Copyright (c) 2017 Uber Technologies, Inc.
```
Comment (yurishkuro) [inline opentracing_instrumentation/client_hooks/__init__.py]: I think you changed it in the wrong place, I meant the new file you're adding. This one can be changed to 2015-2017.

Comment (bitglue) [reply opentracing_instrumentation/client_hooks/__init__.py]: :man_facepalming: fixed

**Pull request merged**